staging/src/k8s.io/sample-apiserver/pkg/admission/plugin/banflunder/admission.go

```diff
@@ -44,7 +45,12 @@ func (d *disallowFlunder) Admit(a admission.Attributes) error {
 		return nil
 	}
 
-	flunderName := a.GetName()
+	metaAccessor, err := meta.Accessor(a.GetObject())
```

Comment:
this is incorrect... on update, the name in the URL governs, and the name in the object has not yet been verified